build.psm1

```diff
@@ -1513,10 +1512,8 @@ function Install-Dotnet {
         }
 
         if ($uninstallScript) {
-            Start-NativeExecution {
-                curl -sO $obtainUrl/uninstall/$uninstallScript
-                Invoke-Expression "$sudo bash ./$uninstallScript"
-            }
+            Start-NativeExecution [scriptblock]::Create("curl -sO $obtainUrl/uninstall/$uninstallScript")
```

Comment:
`[scriptblock]::Create` and then invoking it is equivalent of Invoke-Expression.

Reply:
```suggestion	
            $uninstallScriptUrl = "$obtainUrl/uninstall/$uninstallScript"
            Start-NativeExecution {curl -sO $uninstallScriptUrl}
```